src/Rules/Between.php

```diff
@@ -19,20 +19,18 @@ public function allowedParameters(): array
      *
      * @param mixed $value
      */
-    public function passes($value, array $parameters): bool
+    public function passes($value, array $parameters, array $row): bool
     {
-        $size = (int) $value;
-
         list($min, $max) = $parameters;
 
-        return $size >= (int) $min && $size <= (int) $max;
+        return +$value >= +$min && +$value <= +$max;
```

Comment:
Prefer explicit casting with `(int)`

Reply:
I went with this because I wanted to keep the rules generalised for both floats and ints. If we want an explicit cast then we would need separate rules for int and float. Maybe I can add an additional method like cast($parameters) in the Between rule and extend it for BetweenFloat. Let me know your thoughts on this.

```php
protected function cast(array $parameters): array
{
    return array_map('intval', $parameters);
}
```

Reply:
You are right I wasn't thinking of floats. We can keep it as is